target/ppc/translate/vsx-impl.c.inc

```diff
@@ -2185,6 +2181,49 @@ TRANS(XXBLENDVH, do_xxblendv, MO_16)
 TRANS(XXBLENDVW, do_xxblendv, MO_32)
 TRANS(XXBLENDVD, do_xxblendv, MO_64)
 
+static bool do_xsmaxmin(int t, int a, int b,
```

Comment:
I am thinking if this is the best approach. Should I remove this function and move its body to do_xsmaxmincjdp/do_xsmaxmincqp, changing do_xsmaxmincqp to use gen_avr_ptr?